### PR TITLE
docs/tutorials/podman_tutorial.md

### PR DESCRIPTION
**Pull request by baude**
Update now that more of the podman commands are working
correctly, esp top and friends.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: docs/tutorials/podman_tutorial.md [hunk 1]
```diff
@@ -101,19 +101,16 @@ $ sudo podman ps
...
+the inspect subcommand to see what IP address was assigned to the container.
 ```
-$ sudo podman exec --latest -t dnf -y install iproute
-$ sudo podman exec --latest -t ip a
+$ sudo podman inspect -l | grep IPAddress\":
+        "IPAddress": "10.88.6.140",
 ```
 
-Note the IP address of the *ethernet* device.
+Note: The -l is convenience arguement for **latest container**.  You can also use the container's ID instead
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/podman_tutorial.md]: I'd rather --latest here instead of -l.  But that's me and my ocd.

**Comment by TomSweeneyRedHat** [comment]
Red Hat CI seems unhappy with some unrelated issues.

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by mheon** [comment]
Tests are green, LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by mheon** [comment]
Homu definitely stuck, tests green, docs only change. Merging manually.

**Pull request merged**